internal/pkg/metrics/manifest.go

```diff
@@ -33,6 +33,7 @@ const (
 	ManifestReconcileFinished            ManifestRequeueReason = "manifest_reconcile_finished"
 	ManifestUnmanagedUpdate              ManifestRequeueReason = "manifest_unmanaged_update"
 	ManifestResourcesLabelRemoval        ManifestRequeueReason = "manifest_labels_removal"
+	ManifestOrphaned                     ManifestRequeueReason = "manifest_orphaned"
```

Comment:
Since we were talking about the cleanup yesterday:

Just setting this metric should be fine, we are calling cleanup from the metrics pkg:
`internal/pkg/metrics/manifest.go`